rskj-core/src/main/java/org/ethereum/config/Constants.java

```diff
@@ -46,6 +46,9 @@ public class Constants {
     private static final BigInteger TRANSACTION_GAS_CAP = BigDecimal.valueOf(Math.pow(2, 60)).toBigInteger();
     private static final BigInteger RSKIP156_DIF_BOUND_DIVISOR = BigInteger.valueOf(400);
 
+    private static final long DEFAULT_MAX_TIMESTAMPS_DIFF_IN_SECS = 5L * 60;  // 5 mins
+    private static final long TESTNET_MAX_TIMESTAMPS_DIFF_IN_SECS = 30L * 60; // 30 mins
```

Comment:
this is because they are part of the consensus and should not be modifiable - should only be changed via hard forks (all nodes should apply the change simultaneously)

Reply:
oh true, I could've guessed that one, thanks!